### PR TITLE
[T2A4][W14-A4]Tan Wei

### PR DESCRIPTION
**Pull request by tanwei0319**

### CODE REVIEW

**Comment by tanwei0319** [comment]
Ready for review

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -484,14 +481,37 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
...
+    	Iterator<String> strIter = strings.iterator();
+    	
+    	while(strIter.hasNext()) {
+    		String tempString = strIter.next();
+    		lowercaseStrings.add(tempString.toLowerCase());
+    	}
+    	
+    	return lowercaseStrings;
+    }
+
```
Comment (K1ang) [inline src/seedu/addressbook/AddressBook.java:513]: Good job here! Keep it up the commenting work.

Comment (K1ang) [reply src/seedu/addressbook/AddressBook.java:513]: Next would be to do up the necessary documentations and tests for the modified behavior.

**Review by K1ang — COMMENTED** [review]
@tanwei0319 Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**